packages/terra-text/src/Text.module.scss

```diff
@@ -39,4 +47,11 @@
     position: absolute;
     width: 1px;
   }
+
+  .font-stack {
+    &::after {
+      content: var(--terra-text-v2-font-stack-after-content, '\"Helvetica Neue\", Helvetica, Arial, sans-serif');
```

Comment:
This intended for the theme repos so we can change what [displays for the font-stack in the first example](http://localhost:8080/#/components/terra-text/text/text).